.isort.cfg

```diff
@@ -0,0 +1,3 @@
+[settings]
+line_length=100
+known_third_party = Crypto,IPython,OpenSSL,PIL,_pytest,attr,bottle,bugzilla,cStringIO,cached_property,click,coverage,cycler,dateparser,dateutil,debtcollector,deepdiff,diaper,docker,dockerbot,dump2polarion,fauxfactory,freeze,gevent,git,github,glanceclient,humanfriendly,importscan,iso8601,jenkins,jinja2,jira,jsmin,keystoneauth1,kwargify,lxml,lya,manageiq_client,matplotlib,miq_version,navmazing,novaclient,numpy,paramiko,pathlib2,pdfminer,pika,pkg_resources,py,pygal,pytesseract,pytest,pytz,requests,riggerlib,rope,scp,selenium,sentaku,setuptools,six,slumber,sphinx,sphinx_rtd_theme,sqlalchemy,tabulate,tzlocal,urllib2,wait_for,werkzeug,widgetastic,widgetastic_patternfly,wrapanapi,xmltodict,yaml,yaycl,yaycl_crypt,zmq
```

Comment:
Do we have to hand manage this now if we add new requirements?

Reply:
its pre-commit hook managed